net-libs/serf/serf-1.3.9-r2.ebuild

```diff
@@ -14,7 +14,8 @@ SRC_URI="mirror://apache/${PN}/${P}.tar.bz2"
 LICENSE="Apache-2.0"
 SLOT="1"
 KEYWORDS="~alpha amd64 arm arm64 ~hppa ~ia64 ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc x86 ~x64-cygwin ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x64-solaris"
-IUSE="kerberos static-libs"
+IUSE="kerberos"
```

Comment:
https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html says:
> Examples of changes that can be done without a revision bump are:
> ...
> * adding a new USE flag or removing an existing one (since change in USE flags is going to trigger `--changed-use` rebuild),